gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -354,6 +432,26 @@ public void onCameraMove(CameraPosition newCameraPosition) {
             project -> projectRepository.setCameraPosition(project.getId(), newCameraPosition));
   }
 
+  public void onCompletePolygonButtonClick() {
+    updatePolygonDrawing(PolygonDrawing.STARTED);
```

Comment:
Shouldn't this be `COMPLETED`?

Reply:
removed was not needed as state is already updated in checkPointNearVertex